core/src/main/java/org/testcontainers/utility/MountableFile.java

```diff
@@ -10,7 +10,11 @@
 import org.jetbrains.annotations.NotNull;
 import org.testcontainers.images.builder.Transferable;
 
-import java.io.*;
+import java.io.File;
```

Comment:
my intellij configured to not allow wildicard imports. Dunno what's your policy.